### PR TITLE
add eventlistener to remove customcursor when hovering link

### PR DESCRIPTION
**Pull request by mikael-johnsson**
Eventlistener changes the customCursor and horizontalLine back to default cursor when hovering a link, and changing back when hovering stops.

Eventlisteners removed when ruler checkbox unchecked.

### CODE REVIEW

File: ruler.js [hunk 1]
```diff
@@ -51,18 +76,54 @@ function createRuler() {
...
 
 function removeRuler() {
         // Remove cursor and horizontal line
         const customCursor = document.getElementById("custom-cursor");
         const horizontalLine = document.getElementById("horizontal-line");
+        const parentRulerContainer = document.getElementById("parent-container");
+        const childRulerContainer = document.getElementById("child-container");
+
+        // Remove EventListeners
+        removeEventListener("mouseenter", enterLink);
```
Comment (ShaneByrne0963) [inline ruler.js:94]: This should follow the same rule as the addEventListener. Should be removed from each link

**Review by ShaneByrne0963 — CHANGES_REQUESTED** [review]
removeEventListener should be called on all links

File: ruler.js (hunk 1 above)
Comment (mikael-johnsson) [reply ruler.js]: changes done

**Review by ShaneByrne0963 — APPROVED** [review]
LGTM

**Pull request merged**